### PR TITLE
18951 Change amalgamation to amalgamationApplication

### PR DESCRIPTION
**Pull request by kzdev420**
*Issue #:* /bcgov/entity#18951

*Description of changes:*

Amalgamation Object will be like this
```
filing: {
        header: {
          name: 'amalgamationApplication',   // updated
          accountId: 3040
        },
        business: {
          legalType: 'BC'
        },
        amalgamationApplication: {     // updated
          nameRequest: {
            legalType:  'BC'
          },
          type: 'regular'
        }
      }
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Review by severinbeauvais — APPROVED** [review]

**Comment by severinbeauvais** [comment]
Kevin, you need to import the latest shared enums. That's why the build-check has failed.

Next time, please run the build locally (and also the unit tests and linting) before assigning reviewers to the PR.

**Review by JazzarKarim — APPROVED** [review]

File: src/components/new-request/business-lookup.vue [hunk 1]
```diff
@@ -5,13 +5,13 @@
     id="business-lookup"
   >
     <v-autocomplete
+      v-model:search-input="searchField"
```
Comment (severinbeauvais) [inline src/components/new-request/business-lookup.vue:8]: No, don't change this as it doesn't work in Vue2/Vuetify 2.

I think a lot of these changes were made automatically when you ran lint locally (without the no-fix option). Please revert all the changes except YOURs.

Comment (kzdev420) [reply src/components/new-request/business-lookup.vue:8]: Got it

**Comment by severinbeauvais** [comment]
@eve-git @ozamani9gh We would like to merge this PR as soon as possible. Are you able to review this PR? Can we merge it ourselves?

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://namerequest-dev--pr-750-7lf1pe42.web.app

**Pull request merged**